code/src/App.js

```diff
@@ -1,9 +1,80 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react';
+
+import Form from './components/Form';
+import Header from './components/Header';
+import MessageBoard from './components/MessageBoard';
+
+import {API_URL, API_LIKES} from './reusable/urls';
 
 export const App = () => {
+    // Declaring useStates:
+    const [happyThoughtsList, setHappyThoughtsList] = useState([]);
+    const [messageNew, setMessageNew] = useState("");
+    
+    // useEffect to control when we fetch from API:
+  useEffect(() => {
+    fetchHappyThoughts();
+  }, []);
+
+    // Function to fetch all the thoughts with API:
+    const fetchHappyThoughts = () => {
+        fetch(API_URL)
+        .then(response => response.json())
+        .then(data => setHappyThoughtsList(data))
+        .catch(err => console.error(err));
+    };
+
+    const handleMessageNewChange = (e) => {
+      setMessageNew(e.target.value);
+    };
+
+    const handleFormSubmit = (e) => {
```

Comment:
Good naming of functions! Now it's clear that this one is used in one of the components 😊